fseai/fseai_models.py

```diff
@@ -0,0 +1,54 @@
+"""
+Deep Learning models in fseai
+"""
+import torch.nn as nn
+
+
+class FCmodel(nn.Module):
```

Comment:
Class names are normally written in pascal cases -> FCModel (note the "M" is also in caps)